module/fixture/src/Application/Command/ErgonodeFixtureCommand.php

```diff
@@ -53,6 +56,26 @@ public function configure(): void
      */
     public function execute(InputInterface $input, OutputInterface $output): void
     {
+        $stopwatch = new Stopwatch();
+        $stopwatch->start('ergonode-fixture-load');
         $this->process->process($input->getOption('group'));
+        $event = $stopwatch->stop('ergonode-fixture-load');
+
+        $this->endFixtureLoad($event, $output);
+    }
+
+    private function endFixtureLoad(StopwatchEvent $stopwatchEvent, OutputInterface $output)
```

Comment:
Pleas Repair Code Style 
```suggestion
/**
* @param StopwatchEvent $stopwatchEvent
* @param OutputInterface $output
*/
    private function endFixtureLoad(StopwatchEvent $stopwatchEvent, OutputInterface $output):void
```

Reply:
done